docs/INSTALLATION.md

```diff
@@ -162,6 +163,7 @@ Daarnaast zijn er bronnen die binnen KISS doorzocht moeten worden.
 | `ZAAKSYSTEEM__0__API_CLIENT_ID`         | ClientId voor de ZGW API's                                                                                                       |
 | `ZAAKSYSTEEM__0__DEEPLINK_URL`              | Basisurl om te deeplinken naar een Zaak in het zaaksysteem (optioneel) <details> <summary>Meer informatie </summary> Bijvoorbeeld: `https://zaaksysteem.mijngemeente.nl/mp/zaak/` <br /> Deze variabele **moet** altijd gebruikt worden **in combinatie met** `ZAAKSYSTEEM__0__DEEPLINK_PROPERTY`. Als deze variabelen beiden worden ingevuld, zal er in KISS een link in het zaakdetailscherm staan, waarmee de KCM de betreffende zaak direct in het zaaksysteem opent.  LET OP: dit kan alleen bij zaaksystemen die een vaste url hebben voor zaakdetails, waarbij alleen één property van de zaak, bijv. het zaaknummer, áchter die URL geplaatst hoeft te worden.  </details>                                                                             |
 | `ZAAKSYSTEEM__0__DEEPLINK_PROPERTY`         | Property om naar een zaak te kunnen deeplinken (optioneel) <details> <summary>Meer informatie </summary> Deze variabele **moet** altijd gebruikt worden **in combinatie met** `ZAAKSYSTEEM__0__DEEPLINK_URL`. De waarde uit dit property van een specifieke zaak wordt achter `ZAAKSYSTEEM_DEEPLINK_URL` geplaatst om de link te laten werken. Bijvoorbeeld: `identificatie` </details>  |
+| `ZAAKSYSTEEM__0__NIETNATUURLIJKPERSOONIDENTIFIER`       | Identifier voor 'niet natuurlijke personen' <br/> (`rsin` of `kvkNummer`) voor dit zaaksysteem<details> <summary>Meer informatie </summary> Afhankelijk van de gebruikte bron (bijvoorbeeld Open Zaak of de e-Suite) kan je hiermee aangeven welk gegeven gebruikt wordt om KvK-gegevens van `niet natuurlijke persoon`en aan klanten in het klantregister te koppelen. Op dit moment kan je hiervoor het RSIN (`rsin`) of het KvK-nummer (`kvkNummer`) gebruiken. RSIN is de default: als deze variable leeg gelaten wordt of ontbreekt bij de installatie, zal `rsin` gebruikt worden. Als je de e-Suite als register gebruikt, moet je hier `kvkNummer` invullen. Let op, de spelling moet exact overeen komen.</details>                     |

```

Comment:
de NIETNATUURLIJKPERSOONIDENTIFIER was oorspronkelijk t.b.v. het aanmaken/ vinden van Klanten bij KvK-gegevens uit KvK, waar geen vestigingsnummer is. 
Hij is nu verplaatst naar ZAAKSYSTEEM. 
Twee checkvragen: 
- klopt het idd dat we 'm voor 'Klanten' niet meer nodig hebben. want we gaan de KLANTEN API niet meer ondersteunen?
- De omschrijving moet eigenlijk bijgewerkt worden: het gaat er nu om, om Zaken te kunnen vinden bij bedrijven zonder vestigingsnummer, toch?

Reply:
Dat klopt inderdaad. Ik heb de omschrijving wat verduidelijkt, goed punt. Is het zo beter?

Reply:
Ja, dank :-)